### ISSUE TITLE
Add theme version constant to functions 

### ISSUE THREAD

**Issue by erwstout** [factor1/prelude-wp#84]
Adding a theme version constant to `functions.php` and adding that to CSS and JS enqueues can help with caching issues that may arise. 

```php
define( 'VENDD_VERSION', '1.1.9' );

// example enqueue
wp_enqueue_script( 'vendd-scripts', get_template_directory_uri() . '/inc/js/vendd-scripts.js', array( 'jquery' ), VENDD_VERSION, true );
```

Thanks to Chris Klosowski ( @cklosowski ) for the heads up and example.

**Comment by bebaps** [comment]
I don't see any reason not to do this.

**Comment by erwstout** [comment]
I'll roll it in on Friday

**Comment by erwstout** [comment]
Fixed in `3.3.11`
